docs/source/production.rst

```diff
@@ -96,6 +96,7 @@ Detsim *light tables* (**LTs**) and *point spread functions* (**PSFs**) can be f
 
 .. code-block:: text
 
-  /data5/users/gdiaz/NEXT100/kr83m/LightTables
+  /data5/users/gdiaz/NEXT100/kr83m/LightTables (for productions before May 2024)
```

Comment:
This first path doesn't seem to exist There are tables under `/data5/users/gdiaz/NEXT100/LightTables/`, though. Are they the correct ones?

Reply:
Looks like it was wrong indeed. Fixed now.